sql-parser/src/main/java/io/crate/sql/tree/ColumnElements.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.sql.tree;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class ColumnElements extends Node {
```

Comment:
I'd prefer to align `ColumnDefinition` with `AddColumnDefinition` than adding another layer beneath.
I think this could be done easily by defining 2 ctors:

```
  public ColumnDefinition(String name, ....) {
    this.columnIdent = ColumnIdent.fromNameSafe(name)
  }

  public ColumnDefinition(Expression name, ....) {
    this.columnIdent = ExpressionToColumnIdentVisitor.convert((name)
  }  
```